packages/shared/src/__tests__/keys.test.ts

```diff
@@ -58,6 +58,12 @@ describe('parsePublishableKey(key)', () => {
     expect(() => parsePublishableKey('fake_pk', { fatal: true })).toThrowError('Publishable key not valid.');
   });
 
+  it('throws an error if the publishable key is missing, when fatal: true', () => {
+    expect(() => parsePublishableKey(undefined, { fatal: true })).toThrowError(
+      'Publishable key is missing. Ensure that your publishable key is correctly configured. Make sure to double-check your environment configuration (.env file).',
```

Comment:
```suggestion
      'Publishable key is missing. Ensure that your publishable key is correctly configured. Double-check your environment configuration (e.g. your .env file) for your keys, or access them here: https://dashboard.clerk.com/last-active?path=api-keys',
```

Reply:
Thanks! Dropped the `(e.g. your .env file)` part, because it may confuse users on frameworks that have different env files.